pkg/api/api.go

```diff
@@ -65,6 +70,9 @@ type Config struct {
 
 	// Allows and is used to configure the addition of HTTP Header fields to logs
 	HTTPRequestHeadersToLog flagext.StringSlice `yaml:"http_request_headers_to_log"`
+
+	// This sets the Origin header value
+	corsRegexString	string `yaml:"-"`
```

Comment:
This needs to include `yaml:"cors_origin"` so the flag is `-server.cors-origin` 
The docs need to be generated/included in the PR using make doc

Reply:
updated